notices.json

```diff
@@ -1,4 +1,20 @@
 [
+  {
+    "id": "boards_deprecations",
+    "conditions": {
+      "audience": "sysadmin",
+      "instanceType": "cloud"
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Mattermost Boards will be removed on September 28, 2023",
+        "description": "Mattermost Boards is transitioning to a fully community supported plugin. As a result, Boards will be removed from Mattermost Cloud instances on September 21, 2023. Click below to learn more about how this impacts your instance and how to prepare.",
```

Comment:
@amyblais looks like this date was missed in the description when we sent the notice, can you help update it in a new PR?